core/src/main/java/org/infinispan/remoting/inboundhandler/TrianglePerCacheInboundInvocationHandler.java

```diff
@@ -235,7 +234,7 @@ private BlockingRunnable createIndirectRpcRunnable(SingleRpcCommand command, int
       return new DefaultTopologyRunnable(this, command, Reply.NO_OP, TopologyMode.READY_TX_DATA, commandTopologyId, false) {
          @Override
          protected void onException(Throwable throwable) {
-            sendExceptionAck(writeCommand.getCommandInvocationId(), new RemoteException("Exception on " + localAddress, throwable), commandTopologyId);
+            sendExceptionAck(writeCommand.getCommandInvocationId(), throwable, commandTopologyId);
```

Comment:
Did you have any reason to drop the RemoteException wrapping? I probably found useful having the address of exception source at hand...

Reply:
Except for `OutdatedTopologyException` and `AvailabilityException`, the wrapping still happens.